integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java

```diff
@@ -58,6 +75,63 @@
+  // Get BASE_IMAGES_REPO from env var, if its not provided use OCIR as default to pull base images
+  public static final String BASE_IMAGES_REPO = Optional.ofNullable(System.getenv("BASE_IMAGES_REPO"))
+      .orElse(OCIR_DEFAULT);
+  // Use OCR secret name if OCR is used for base images, if not use OCIR secret name
+  public static final String BASE_IMAGES_REPO_SECRET =
+      BASE_IMAGES_REPO.equals(OCR_REGISTRY) ? OCR_SECRET_NAME : OCIR_SECRET_NAME;
+
+  // Get WEBLOGIC_IMAGE_NAME/WEBLOGIC_IMAGE_TAG from env var, if its not provided and
+  // if base images repo is OCR use OCR default image values
+  // or if base images repo is OCIR use OCIR default image values
+  public static final String WEBLOGIC_IMAGE_NAME
+          = BASE_IMAGES_REPO + "/" + Optional.ofNullable(System.getenv("WEBLOGIC_IMAGE_NAME"))
+      .orElse(BASE_IMAGES_REPO.equals(OCR_REGISTRY) ? OCR_WEBLOGIC_IMAGE_NAME : OCIR_WEBLOGIC_IMAGE_NAME);
+  public static final String WEBLOGIC_IMAGE_TAG = Optional.ofNullable(System.getenv("WEBLOGIC_IMAGE_TAG"))
+      .orElse(BASE_IMAGES_REPO.equals(OCR_REGISTRY) ? OCR_WEBLOGIC_IMAGE_TAG : OCIR_WEBLOGIC_IMAGE_TAG);
+  public static final String WLS_UPDATE_IMAGE_TAG = "14.1.1.0-11";
+
+  // Get FMWINFRA_IMAGE_NAME/FMWINFRA_IMAGE_TAG from env var, if its not provided and
+  // if base images repo is OCR use OCR default image values
+  // or if base images repo is OCIR use OCIR default image values
+  public static final String FMWINFRA_IMAGE_NAME
+      = BASE_IMAGES_REPO + "/" + Optional.ofNullable(System.getenv("FMWINFRA_IMAGE_NAME"))
+      .orElse(BASE_IMAGES_REPO.equals(OCR_REGISTRY) ? OCR_FMWINFRA_IMAGE_NAME : OCIR_FMWINFRA_IMAGE_NAME);
+  public static final String FMWINFRA_IMAGE_TAG = Optional.ofNullable(System.getenv("FMWINFRA_IMAGE_TAG"))
+      .orElse(BASE_IMAGES_REPO.equals(OCR_REGISTRY) ? OCR_FMWINFRA_IMAGE_TAG : OCIR_FMWINFRA_IMAGE_TAG);
+
+  // Get DB_IMAGE_NAME/DB_IMAGE_TAG from env var, if its not provided and
+  // if base images repo is OCR use OCR default image values
+  // or if base images repo is OCIR use OCIR default image values
+  public static final String DB_IMAGE_NAME
+      = BASE_IMAGES_REPO + "/" + Optional.ofNullable(System.getenv("DB_IMAGE_NAME"))
+      .orElse(BASE_IMAGES_REPO.equals(OCR_REGISTRY) ? OCR_DB_IMAGE_NAME : OCIR_DB_IMAGE_NAME);
+  public static final String DB_IMAGE_TAG = Optional.ofNullable(System.getenv("DB_IMAGE_TAG"))
+      .orElse(BASE_IMAGES_REPO.equals(OCR_REGISTRY) ? OCR_DB_IMAGE_TAG : OCIR_DB_IMAGE_TAG);
+
+  // For kind, replace repo name in image name with KIND_REPO, otherwise use the actual image name
+  // For example, image container-registry.oracle.com/middleware/weblogic:12.2.1.4 will be pushed/used as
+  // localhost:5000/middleware/weblogic:12.2.1.4 in kind and in non-kind cluster it will be used as is.
+  public static final String WEBLOGIC_IMAGE_TO_USE_IN_SPEC = KIND_REPO != null ? KIND_REPO
```

Comment:
Can you change this to WEBLOGIC_IMAGE_LOCAL as we discussed yesterday.

Reply:
sure, I will change in next PR as I have to do the testing again.